webpaste.el

```diff
@@ -620,6 +628,13 @@ Takes optional POINT and MARK to paste a region."
     ;; Else, Paste buffer
     (webpaste-paste-buffer)))
 
+
+(cl-defun webpaste--interactive-get-provider ()
```

Comment:
This added function doesn't seem to be... used? It's not autoloaded either, so I don't really see the use for it.

Reply:
gah, that wasn't supposed to get added, it's part of a separate change, removed now